<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialog</class>
 <widget class="QDialog" name="AddItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Изменение элемента</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ст. название: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="nameMass"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelQuantity">
       <property name="text">
        <string>Количество:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="quantityInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPrice">
       <property name="text">
        <string>Цена:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="priceInput">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelDescription">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="descriptionInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
